dcp_prototype/backend/wrangling/migrations/common/gather_dcp_one_data.py

```diff
@@ -176,7 +176,6 @@ def generate_metadata_structure_from_dir(input_dir, num_threads, dataset_metadat
 
 
 def generate_metadata_structure_from_targz(input_source, dataset_metadata):
-    index = 0
```

Comment:
index is used in line  194,195.  It's used to print the number of files accessed.

Reply:
This variable is defined/incremented in the for loop in 188 though so this is a redundant initiation.

Reply:
Yes, you're right.